package/native-package/src/index.js

```diff
@@ -247,6 +249,24 @@ registerNativeHandlers({
       ignoreAndroidSystemSettings: false,
     });
   },
+  // eslint-disable-next-line react/display-name
```

Comment:
Should we add a task to add a displayname to the video component?

Reply:
As Mads said, this is NOT blocking for the release: @khushal87 let's solve this by creating a const arrow func or any other valid solution besides suppressing the linter